helm_deploy/values-prod.yaml

```diff
@@ -106,6 +106,7 @@ generic-service:
     USER-ALLOCATIONS_RULES_EMERGENCY-AND-SHORT-NOTICE-ASSESSMENTS_ALLOCATE-TO-USERS_WALES: KEVINPENNY1NPS
 
     FEATURE-FLAGS_CAS1-APPEAL-MANAGER-CAN-ASSESS-APPLICATIONS: true
+    FEATURE-FLAGS_CAS1_ONLY_LIST_CONVICTION_DOCUMENTS: true
```

Comment:
this should be false until testing is complete

we don't appear to have a value added for pre-prod - was this intended for there instead?